app/assets/stylesheets/components/ui/_cookie.scss

```diff
@@ -15,26 +15,14 @@
 }
 
 .cookie-policy-content {
-  padding: 0 $baseline-spacing 0 0;
-  margin: 0 5px 0 0;
-  font-size: 0.875em;
+  @include column(12);
```

Comment:
I'm pretty sure this is unnecessary: in FinCap we default to the equivalent of `column(12)`